src/ts_generator/sql_parser/translate_stmt.rs

```diff
@@ -48,12 +52,15 @@ pub async fn translate_stmt(
             from,
             order_by: _,
             limit: _,
-        } => {
-            let table_name = get_default_table(from);
-            let table_name = table_name.as_str();
-            let selection = selection.to_owned().unwrap();
-            translate_delete(ts_query, &selection, table_name, db_conn).await?;
-        }
+        } => match &from {
+            FromTable::WithFromKeyword(from) => {
+                let table_name = get_default_table(from);
+                let table_name = table_name.as_str();
+                let selection = selection.to_owned().unwrap();
+                translate_delete(ts_query, &selection, table_name, db_conn).await?;
+            }
+            FromTable::WithoutKeyword(_) => Err(TsGeneratorError::FromWithoutKeyword(sql_statement.to_string()))?,
```

Comment:
Note: impossible to add a test for this because this particular SQL syntax only exists in Google bigquery